.travis.yml

```diff
@@ -17,13 +17,19 @@ cache:
     directories:
         - $HOME/.composer/cache
 
+before_script:
+    - echo 'phar.readonly = "0"' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
 before_install:
     - sudo add-apt-repository -y ppa:git-core/ppa
     - sudo apt-get update && sudo apt-get install git-man && sudo apt-get install -y git
     - git --version
     - phpenv config-rm xdebug.ini || true
+    - export PATH="$PATH:$HOME/.composer/vendor/bin"
 
-install: composer update --prefer-dist --no-interaction --optimize-autoloader
+install:
+    - composer global require kherge/box --prefer-dist --no-interaction --optimize-autoloader
```

Comment:
@chalasr : Ideally, this dep should be in the `require-dev` section of our `composer.json`, as we need it in order to the test suite to work (or at least we should mark the test as incomplete/skipped if box2 is not available on the host and only enforce this on the Travis side). 

But...I'm unable to require it, and can't figure out why... It seems its only looking for symfony `~2.1` components,  despite the constraints looks good to me: https://github.com/box-project/box2/blob/2.7.4/composer.json

Do you have any hint ? 😕